docs/flocks/actions.md

```diff
@@ -201,7 +394,57 @@ You cannot delete the [Default Flock](/guide/terminology.html#default-flock), bu
 
 ::::: slot description
 
-Move a specified [Sensor](/guide/terminology.html#sensors) to a Flock. You can optionally delete all incidents before moving the Sensor by specifying `clear_incidents=true` (this would usually be recommended to ensure sensitive information doesn't get spilled across Flocks).
+You'll need to delete all incidents before moving a [Sensor](/guide/terminology.html#sensors) between Flocks (this ensures that sensitive data isn't spilled across Flocks). If there are still incidents tied to the Sensor, you can specify `clear_incidents=true` to clear all of those incidents.
```

Comment:
Let's keep the opening sentence: `Move a specified [Sensor](/guide/terminology.html#sensors) to a Flock.`